live-build/includes.chroot/usr/local/bin/flash-image.sh

```diff
@@ -738,10 +848,20 @@ fi
 # NOTE: This is a one time change that will not affect the stored boot order
 efibootmgr -n ${new_entry} > /dev/null 2>&1
 
+# todo: make this more dynamic?
+# todo: put it in a function triggered on any script exit?
+mount /dev/$data /mnt
+flash_log="${install_date}-${boot_label}-install.log"
```

Comment:
maybe do a full timestamp so a few installs in a day won't overwrite each other?